### ISSUE TITLE
jquery-ui installed in wrong location?

### ISSUE THREAD

**Issue by wagnered** [yiisoft/yii#3703]
I installed yiisoft/yii2-jui with composer.  It installed jquery-ui in bower-asset, but when I set up to use the Dialog, I got an error stating:

"The file or directory to be published does not exist: ~/htdocs/_protected/vendor/bower/jquery-ui".

When I copied the folder into the bower folder, the error disappeared.

I don't know if that was the correct work-around or if it will cause  other problems, but I thought I'd report what I experienced.

**Comment by samdark** [comment]
Have you installed Composer plugin globally as noted in installation docs?

**Comment by samdark** [comment]
btw., it's Yii 1.1 issue tracker.

**Comment by wagnered** [comment]
On Sat, 2015-01-10 at 23:00 -0800, Alexander Makarov wrote:
> Have you installed Composer plugin globally as noted in installation
> docs?
> 
> —
> Reply to this email directly or view it on GitHub.
> 


This is what I used to install composer:


> curl -s http://getcomposer.org/installer | php
> mv composer.phar /usr/local/bin/composer

from the Yii2 install doc.

It's been awhile, so I re-ran:

"composer global require "fxp/composer-asset-plugin:1.0.0-beta4""

again and it indicated that it removed the original installation and
re-installed again.

So the answer is "yes".

But I did not notice that this is the 1.1 issue tracker as my eyesight
is not so good.

I'll take it over to the 2.0 tracker.

Regards...

-- 
73 From Ernie D

AA1AD | CN87ug
